<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAMP Plugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="button_start_samp">
     <property name="text">
      <string>Start SAMP</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_status">
       <property name="text">
        <string>&lt;current status&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QCheckBox" name="bool_highlight_is_selection">
     <property name="text">
      <string>Treat row highlighting in other applications as a selection</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="button_stop_samp">
     <property name="text">
      <string>Stop SAMP</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
